index.html

```diff
@@ -73,6 +73,229 @@ <h1>I'm Chris. Glad to see you!</h1>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="card--image"
+            src="images/project3.png"
+            alt="Tonic project image"
+          />
+          <div class="card--body">
+            <h2>Tonic</h2>
+            <div class="card--body-details">
+              <h4>CANOPY</h4>
+              <div>
+                <ul class="card--body-list">
+                  <li class="card--body-role">Back End Dev</li>
+                  <li class="card--body-year">2015</li>
+                </ul>
+              </div>
+            </div>
+            <p class="card--body-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required
+            </p>
+          </div>
+          <div class="card--languages">
+            <button class="btn--primary">html</button>
+            <button class="btn--primary">css</button>
+            <button class="btn--primary">javascript</button>
+          </div>
+          <div class="card--project-btn">
+            <a href="#" class="btn--secondary">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="card--image"
+            src="images/project4.png"
+            alt="Multi-Post Stories project image"
+          />
+          <div class="card--body">
+            <h2>Multi-Post Stories</h2>
+            <div class="card--body-details">
+              <h4>CANOPY</h4>
+              <div>
+                <ul class="card--body-list">
+                  <li class="card--body-role">Back End Dev</li>
+                  <li class="card--body-year">2015</li>
+                </ul>
+              </div>
+            </div>
+            <p class="card--body-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required
+            </p>
+          </div>
+          <div class="card--languages">
+            <button class="btn--primary">html</button>
+            <button class="btn--primary">css</button>
+            <button class="btn--primary">javascript</button>
+          </div>
+          <div class="card--project-btn">
+            <a href="#" class="btn--secondary">See Project</a>
+          </div>
+        </div>
+      </section>
+      <section class="about--myself">
+        <h2>About Myself</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <a href="#"><h3>LET'S CONNECT</h3></a>
+        <ul class="headline__list">
+          <li class="headline__list_item">
+            <a href="https://www.linkedin.com/in/chris-mutugi-46594b123/"
+              ><svg
+                class="headline__list_item_icon icon"
```

Comment:
- [ ] I think it would be better also to follow the Figma design by adjusting the Typography in the **about section title** You may try to break line as given by Figma template ✨✨

- [ ]  It would be better also to add a ```border-top-right-radius:``` property to match the given Figma design and also to make a beautiful design as well 😎✨
  You may follow this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/border-top-right-radius) for guidance on CSS border-radius 


**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/219617833-9b2e0f3e-4d7a-4747-8426-de3b21ffe838.png) | ![image](https://user-images.githubusercontent.com/102757126/219436751-d4f21cfb-17ef-4571-823d-87fdb6d3dc0f.png)

Reply:
Hello @TuyishimireEric. Thanks for pointing these things out. I had missed some entirely. Kindly check if everything is better now.